src/components/Section/Relationships/Relatives/Relatives.jsx

```diff
@@ -75,6 +100,7 @@ export class Relatives extends Subsection {
   validRelations = () => new RelativesValidator(this.props).validMinimumRelations()
```

Comment:
This seems like a good opportunity to deprecate some usage of the `RelativesValidator` class. Both `validMinimumRelations` and `validMaritalRelations` could be rewritten as functions.